mealplanner-ui/src/pages/MealPlans/MealPlanCard.tsx

```diff
@@ -163,6 +166,11 @@ export const MealPlanCard = (props: MealPlanCardProps) => {
             title={mealplan.nameEn}
             subheader={mealplan.person?.fullName}
           />
+          {startDate && (
+            <div style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
+              <Chip label={"Starting: " + startDate.format("DD-MMM-YYYY")} />
```

Comment:
In hindsight, since 'Starting on' is correct grammatically and 'Starting' sounds incomplete, I thought we could just have it as 'Start Date'. Moreover, don't you think that having it as a tooltip on hover would make it look slicker? That is just the date alone to be shown on the Chip, and on hover, the text 'Start Date' is shown as a tooltip.

Reply:
That's a great idea. thank you.